topics/ui/controls/tabs.md

```diff
@@ -24,49 +25,71 @@ Follow the rules in [Groups of controls](groups_of_controls.md).
 
 ## How to use
 
-Use title capitalization for tab labels.
+### Label
 
-Make the label short, preferably no more than 3 words.
+Use title capitalization for tab labels. Make the label short, preferably no more than 3 words.
 Avoid generic words, such as "General" or "Advanced".
 See [Writing short and clear](writing_short.md).
 
-![](02_naming.png){width=284}
+<table style="none" border="false">
+  <tr>
+<td width="50%">
+      <format color="Green" style="bold">Correct</format><img src="02_How_to_use_correct.png" alt="A correct example of a tab label"/>
+    </td>
+    <td width="50%">
+      <format color="Red" style="bold">Incorrect</format><img src="02_How_to_use_incorrect.png" alt="An incorrect example of a tab label"/>
+    </td>
+  </tr>
+</table>
+
+###  Organization and Accessibility
 
 Place the most frequently used content in the first tab.
+
 Tabs that do not fit allotted screen space automatically hide under the dropdown component.
 (It is better to add no more than 8 tabs, but this number is not limited.)
```

Comment:
I'd suggest removing the round brackets here, just leave the sentence without them.